Pipfile

```diff
@@ -24,10 +24,16 @@ whitenoise = "*"
 django = "*"
 cloudinary = "*"
 faas-cache-dict = "*"
+tzdata = "*"
+python-telegram-bot = "*"
+python-dotenv = "*"
 
 [dev-packages]
 black = "*"
 mypy = "*"
 
 [requires]
 python_version = "3.10"
+
+[pipenv]
+allow_prereleases = true
```

Comment:
did have to do this

Reply:
yes, the bot library is a pre-release edition, but honestly, I had to do it because the stable library syntax is extremely different and migrating to the new syntax will be troublesome.

Reply:
well if everything working fine, that's good